mmcv/device/ipu/ipu_wrapper.py

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+
+
+def is_ipu():
+    try:
+        import poptorch  # noqa: E261, F401
+        IS_IPU = True
+    except ImportError:
+        IS_IPU = False
+    return IS_IPU
+
+
+IS_IPU = is_ipu()
```

Comment:
Move this function and `IS_IPU` to utils.py

Reply:
if put IS_IPU to utils.py, all the codes of utils.py will be placed in a if block(if IS_IPU)